.github/workflows/update-valkey-hashes.yml

```
@@ -0,0 +1,78 @@
+name: Commit new hash into valkey-hashes
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: The version of Valkey to create.
+        type: string
+        required: true
+    secrets:
+      PAT_TOKEN:
+        description: PAT token for valkey-hashes repo.
+        required: true
+
+jobs:
+  commit-hash:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Validate inputs
+        run: |
+          if [[ ! "${{ inputs.version }}" =~ ^([0-9]+\.[0-9]+\.[0-9]+|unstable)$ ]]; then
+            echo "Invalid version format '${{ inputs.version }}'. Expected format: x.y.z or 'unstable'."
+            exit 1
+          fi
+
+      - uses: actions/checkout@v4
+        with:
+          repository: valkey-io/valkey-hashes
```

```suggestion
          repository: ${github.repository_owner}/valkey-hashes
```

We can solve it with this where the change can be tested in your own repository.
Try if this works, I think it should as you are providing your accounts PAT